docs/v3.x/concepts/queries.md

```diff
@@ -353,9 +353,41 @@ const result = await strapi
 const fields = result.toJSON();
 ```
 
+### Knex
+
+Documentation: [http://knexjs.org/#Builder](http://knexjs.org/#Builder)
+
+You can access the Knex instance with 
+
+```js
+const knex = strapi.connections.default;
+```
+
+You can then use Knex to build your own custom queries to the DB. You will lose all the functionalities of the model, 
+but this could come handy if you are building a more custom schema.
+Please note that if you are using the draft system, Strapi nullyfies all the Draft columns util they are published.
```

Comment:
Can you add a linkback to the draft and publish page inside of concepts here?

Reply:
sure, done